docs/contribute/intro.md

```diff
@@ -44,19 +44,20 @@ code looks correct according to a few checks.
 
 ### Run the tests
 
-For information about running tests, see {ref}`developer/tests`.
+For code tests, jupyter-book uses [pytest](https://docs.pytest.org)).
+You may run the tests with the following command:
 
-(developer/tests)=
-## Testing infrastructure
+```shell
+>> pytest --ignore=tests/test_pdf.py
+```
 
-Jupyter Book uses [**`pytest`**](https://docs.pytest.org/en/latest/) for its testing
-infrastructure. You may run the tests with the following command:
+You can alternatively use [tox](https://tox.readthedocs.io), to run the tests in multiple isolated environments, and also without the need for the initial dependencies install (see the `tox.ini` file for available test environments and further explanation):
 
-```bash
-pytest --ignore=tests/test_pdf.py
+```shell
+>> tox -e py38-sphinx3 --  --ignore=tests/test_pdf.py
 ```
 
-This will run the Jupyter Book test suite, *except for the PDF tests*. This is because
+Either will both run the Jupyter Book test suite, *except for the PDF tests*. This is because
```

Comment:
Just being pedantic here :sweat_smile: . Should we start `This is because` in a new line?

Reply:
Yeh not going through and "fixing" all the one-line pre sentence issues 😆 but I have done a bit of tidying in 8511e86, including this